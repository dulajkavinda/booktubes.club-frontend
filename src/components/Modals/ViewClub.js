import * as React from "react";
import Box from "@mui/material/Box";
import Button from "@mui/material/Button";
import Typography from "@mui/material/Typography";
import Modal from "@mui/material/Modal";
import TextField from "@mui/material/TextField";
import MenuItem from "@mui/material/MenuItem";

const style = {
  position: "absolute",
  top: "50%",
  left: "50%",
  transform: "translate(-50%, -50%)",
  width: 400,
  bgcolor: "background.paper",

  boxShadow: 24,
  p: 4,
};

export default function ViewClub({ open, setOpen }) {
  const handleClose = () => {
    setOpen(false);
  };

  const [msg, setMsg] = React.useState("");

  return (
    <div>
      <Modal open={open} onClose={handleClose}>
        <Box sx={style}>
          <TextField
            fullWidth
            label="Message"
            id="fullWidth"
            onChange={(e) => setMsg(e.target.value)}
          />
          <Box height={10} />

          <Button onClick={handleClose} fullWidth variant="contained">
            Post
          </Button>
        </Box>
      </Modal>
    </div>
  );
}
