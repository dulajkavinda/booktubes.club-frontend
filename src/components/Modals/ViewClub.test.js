import React from "react";
import { shallow, mount } from "enzyme";

import ViewClub from "./ViewClub";

let wrapper;
beforeEach(() => {
  wrapper = shallow(<ViewClub />);
});

describe("view club basic tests", () => {
  it("is snapshot changed", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
