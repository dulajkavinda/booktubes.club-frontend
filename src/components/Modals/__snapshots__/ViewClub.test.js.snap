// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view club basic tests is snapshot changed 1`] = `
<div>
  <ForwardRef(Modal)
    onClose={[Function]}
  >
    <ForwardRef(Box)
      sx={
        Object {
          "bgcolor": "background.paper",
          "boxShadow": 24,
          "left": "50%",
          "p": 4,
          "position": "absolute",
          "top": "50%",
          "transform": "translate(-50%, -50%)",
          "width": 400,
        }
      }
    >
      <ForwardRef(TextField)
        fullWidth={true}
        id="fullWidth"
        label="Message"
        onChange={[Function]}
      />
      <ForwardRef(Box)
        height={10}
      />
      <ForwardRef(Button)
        fullWidth={true}
        onClick={[Function]}
        variant="contained"
      >
        Post
      </ForwardRef(Button)>
    </ForwardRef(Box)>
  </ForwardRef(Modal)>
</div>
`;
